src/Shared/PackageManagers/NPMProjectManager.cs

```diff
@@ -158,7 +158,7 @@ public override async Task<IEnumerable<string>> EnumerateVersions(PackageURL pur
                 string? packageName = purl.Namespace != null ? $"@{purl.Namespace}/{purl.Name}" : purl.Name;
                 HttpClient httpClient = CreateHttpClient();
 
-                string? content = await GetHttpStringCache(httpClient, $"{ENV_NPM_API_ENDPOINT}/{packageName}");
+                string? content = await GetHttpStringCache(httpClient, $"{ENV_NPM_API_ENDPOINT}/{packageName}", false);
```

Comment:
Whenever using hard-coded booleans, please include the parameter name: i.e. `useCache: false`